packages/components/src/Dropzone/sub-components/DropzoneError/DropzoneError.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react"
+import { ErrorCode, FileError, FileRejection } from "react-dropzone";
+import { InlineNotification } from "@kaizen/notification";
+import { getKaizenDropzoneError, prettifyErrorCode } from "../../utils";
+
+export const DropzoneError = ({ code, message }: FileError): JSX.Element => {
+  /**  updating this specific error code because the language is awkward. Default code from react-dropzone is 'file-invalid-type'. The other error codes are fine. */
+  const errorCode = code === ErrorCode.FileInvalidType ? "incorrect-file-type" : code
+  const notificationTitle = prettifyErrorCode(errorCode)
+
+  return (
+    <InlineNotification
```

Comment:
Please replace all `InlineNotification` with `FieldMessage` as this is what we use in our form fields.
https://cultureamp.design/storybook/?path=/docs/components-form-subcomponents-field-message--default-kaizen-site-demo